submissions/damilolaSalami/css/signUp.css

```diff
@@ -0,0 +1,407 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    }
+
+/* header */
+.header{
+    background-color: purple;
+    align-items: center;
+    justify-content: space-between;
+    display: flex;
+    padding-top: 0;
+}
+
+.header .logo img{
+    height: 75px;
+    width: 200px;
+}
+
+.header li{
+    display: inline-block;
+    list-style-type: none;
+    font-size: 17px;
+    margin-right: 100px;
+}
+
+.header li a{
+    text-decoration: none;
+    color: white;
+}
+
+.header li a:hover{
+    color: black;
+}
+
+/* Registration */
+.mainbody{
+    display: flex;
+}
+
+.container{
+    flex-basis: 50%;
+}
+
+.registration{
+    font-size: 18px;
+    margin: 150px 100px 0 180px;
+    padding: 0 30px;
+}
+
+.registration h1, .registration a{
+    color: purple;
+    font-weight: bolder;
+}
+
+.regform{
+    padding: 25px 50px 10px 35px;
+    margin: 65px 120px 40px 0;
+    width: 500px;
+    border: 1px solid black;
+    border-radius: 10px;
+}
+
+.regform .inputbox{
+    border: 1px solid black;
+    border-radius: 5px;
+    width: 420px;
+    margin: 4px 1px;
+    height: 24px;
+}
+
+.regform label{
+    font-size: 15px;
+}
+
+.regform .box{
+    margin-left: 80px;
+}
+
+.regform p{
+    color: purple;
+    display: inline;
+    font-weight: bold;
+}
+
+.regform .signup{
+    background-color: purple;
+    font-size: 12px;
+    color: white;
+    border: 1px solid purple;
+    border-radius: 5px;
+    padding: 8px 40px;
+    margin-left: 160px;
+    font-weight: bold;
+}
+
+/* footer */
+.footer{
+    background-color: purple;
+}
+
+.mainfooter{
+    display: flex;
+    width: 100%;
+}
+
+.footerdiv{
+    flex-basis: 50%;
+    padding: 25px 10px 0 40px;
+}
+
+.contact li{
+    list-style-type: none;
+    font-size: 15px;
+    line-height: 25px;
+    font-family: 'Arial';
```

Comment:
@queensaldam Don't set font-family again in the elements you have it in the body already. Having it in the body means, every single element that the body wraps will have that same font-family.